AirCasting/CreateSessionViews/ChooseLocationPickerService.swift

```diff
@@ -6,11 +6,11 @@ import Resolver
 import SwiftUI
 
 class ChooseLocationPickerService: PlacePickerService {
-    @InjectedObject private var tracker: LocationTracker
+    @Injected private var tracker: LocationTracker
     @Binding private var address: String
     
     init(address: Binding<String>) {
-        self._address = address
+        self._address = .init(projectedValue: address)
```

Comment:
What's the reason for using here `projectedValue` instead of `wrppedValue`? Is it a read-only variable?

Reply:
I really have not deep dove into it. Paweł once said that it should be done this way

Reply:
I will let you know in the near future when I will inspect it